cmd/opentelemetry-collector/main.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"github.com/open-telemetry/opentelemetry-collector/config"
+	"github.com/open-telemetry/opentelemetry-collector/exporter"
+	"github.com/open-telemetry/opentelemetry-collector/exporter/zipkinexporter"
+	"github.com/open-telemetry/opentelemetry-collector/processor"
+	"github.com/open-telemetry/opentelemetry-collector/processor/batchprocessor"
+	"github.com/open-telemetry/opentelemetry-collector/receiver"
+	"github.com/open-telemetry/opentelemetry-collector/receiver/zipkinreceiver"
+	"log"
+
+	"github.com/open-telemetry/opentelemetry-collector/service"
+)
+
+func main() {
+	handleErr := func(err error) {
+		if err != nil {
+			log.Fatalf("Failed to run the service: %v", err)
+		}
+	}
+
+	info := service.ApplicationStartInfo{
+		ExeName:  "jaeger-opentelemetry-collector",
+		LongName: "Jaeger OpenTelemetry Collector",
+		// TODO
+		//Version:  version.Version,
+		//GitHash:  version.GitHash,
+	}
+
+	// TODO caused compilation issues due to refactoring in https://github.com/jaegertracing/jaeger/pull/2060
+	// TODO app/HandlerConfig moved to app/handler/HandlerConfig
+	//cmpts, err := defaults.Components()
```

Comment:
I could not initialize all default components (importers, processors, exporters) because Jaeger importer imports jaeger/collector/app and there was a breaking change https://github.com/jaegertracing/jaeger/pull/2060

Reply:
Once https://github.com/open-telemetry/opentelemetry-collector/pull/570 is merged we should be able to use `defaults.components()`

Reply:
I commented on https://github.com/open-telemetry/opentelemetry-collector/pull/570 and we can merge it quickly if it blocks you.